api/schema/resolvers/ContributorResolver.js

```diff
@@ -1,8 +1,39 @@
 const { AuthenticationError } = require('apollo-server')
+const { col, fn } = require('sequelize')
+
 const toggl = require('../../handlers/toggl')
 
 module.exports = {
     Contributor: {
+        paid_by_currency: async (contributor, args, { models }) => {
```

Comment:
I'm not sure how strict this might be but I would recommend keeping consistent with custom properties as camelCase as `paidByCurrency` 

I use this pattern often and it is useful because when you see `snake_case` properties anywhere in the code you can have high confidence it is a column in the DB whereas if you see a `camelCase` property it is a clue that it is not a DB column and is instead a custom function or derived property